homeassistant/components/kulersky/light.py

```diff
@@ -66,7 +66,7 @@ class KulerskyLight(LightEntity):
     def __init__(self, light: pykulersky.Light) -> None:
         """Initialize a Kuler Sky light."""
         self._light = light
-        self._available = None
+        self._available: bool = False
```

Comment:
`None` is not allowed for `available` property. I have replaced it with the corresponding falsy.

Reply:
```suggestion
        self._available = False
```

Reply:
I've dropped the unncessary type hint.